packages/schematics/angular/guard/schema.json

```diff
@@ -49,9 +49,13 @@
       "items": {
         "enum": ["CanActivate", "CanActivateChild", "CanDeactivate", "CanLoad", "CanMatch"],
         "type": "string"
-      },
-      "default": ["CanActivate"],
```

Comment:
This looks like a breaking change.

Reply:
Since this has a default but `implements` and `type` cannot be used together, how should this be handled? Should the logic just change to use `type` when both are provided instead of throwing an error?

Reply:
@dgp1130 suggested implementing the default in the `index.ts`. I think that works, but unresolve this thread if not.